src/context/themeContext.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {setTheme} from '../actions/themeActions';
```

Comment:
Why we need this? we dont have theme anymore in store...
We need to specify callback for provider in App.ts